### ISSUE TITLE
My own IP is marked as

### ISSUE THREAD

**Issue by eyal282** [Munzy/blackbox#719]
__Q: What are you currently seeing?__

The site is marking my IP address as a VPN connection ( Y )?

__Q: What should it be marked as?__

The site should mark it as a non-VPN connection ( N )?

Q:  What is the IP Address?

45.87.212.114

__Q: What is the internet provider?__

Partner

__Q: What other info can you provide us?__

Nothing.

**Comment by Munzy** [comment]
This IP looks to be coming from https://asn.ipinfo.app/AS9009. Is this IP being used on a server or cloud VM?

**Comment by eyal282** [comment]
It's just my own IP.

**Comment by Munzy** [comment]
https://bgp.he.net/ip/45.87.212.114 also reports this being hosted out of AS9009.

Is this your home internet IP address? Is this attached to a server? 

When you say it is "just my own IP." are you saying you own the IP address space? 

From everything I am seeing this looks to be a server IP address.

**Comment by eyal282** [comment]
I mean that I pay the ISP provider and he gave me that IP ( I think that would be accurate, as IP addresses are "created" from my Internet Router )

**Comment by Munzy** [comment]
Something is very weird about this. 

I am seeing that this IP space was used previously for hosting in some capacity. The announcement for this IP Space is a large cloud provider, which is what is flagging you as "Y". I can't find anything that indicates this has been recently swapped over. 

It's almost as if your internet provider is using some sort of VPN to give you an IP address. 

Can you give me the website of your internet provider so I can investigate more? Do you live in Frankfurt Germany?

**Comment by eyal282** [comment]
Okay, I am sorry for the misunderstanding. Here's what happened:

1. I used a VPN to check how good your service is.
2. I disabled the VPN, and went to https://whatismyipaddress.com/
3. The website showed "Tel Aviv / Partner" in the information, but used the IP from the VPN as the data.
4. I don't actually know my IP, but when it showed Tel Aviv and Partner, I thought it was mine.
5. I went on to here.

The thing is that somehow for a little time it showed both the VPN IP I had AND my actual ISP in that website. I don't understand how it could happen, but now that I used my actual IP it properly returned "N"
